eslint.config.mjs

```diff
@@ -13,7 +14,9 @@ const filename = fileURLToPath(import.meta.url);
 const dirname = path.dirname(filename);
 
 export default defineConfig(
+  createJsonConfig(),
```

Comment:
Can you verify if this also include files inside `node_modules` or other ignored directories, since the ignore logic is added in `createBaseConfig()` ?